wiz-kubernetes-connector/templates/_helpers.tpl

```diff
@@ -230,6 +230,6 @@ delete-kubernetes-connector
 {{- if .Values.global.isFedRamp -}}
 publicregistryfedrampwizio.azurecr.us/wiz-app/wiz-broker-fips:{{ .Values.image.tag | default .Chart.AppVersion }}
 {{- else -}}
-{{ coalesce .Values.global.image.registry .Values.image.registry }}/{{ coalesce .Values.global.image.repository .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
+{{ coalesce .Values.global.image.registry .Values.image.registry }}/{{ coalesce .Values.global.image.repository .Values.image.repository }}:{{ coalesce .Values.global.image.tag .Values.image.tag | default .Chart.AppVersion }}
```

Comment:
@liorschach, yay tag is fixed but the coalesce on registry and repository is pointless because [they're not empty](https://github.com/wiz-sec/charts/blob/377b8efa0470a0f26ef5b033781b6916d9bbc1ff/wiz-kubernetes-connector/values.yaml#L183-L184), which is forcing the use of global unnecessarily 😭